Cargo.toml

```
@@ -282,4 +284,21 @@ opt-level = 3
 
 [patch.crates-io]
 # Temporary patch until our PRs are merged.
-tar = { git = "https://github.com/qdrant/tar-rs", branch="main" }
+tar = { git = "https://github.com/qdrant/tar-rs", branch = "main" }
+
+[package.metadata.deb]
+maintainer = "Qdrant Team <team@qdrant.com>"
+depends = "$auto"
+features = ["deb"]
```

This is the default, but it makes it a bit more explicit for me that that default features are used as well.

```suggestion
default-features = true
features = ["deb"]
```